internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestClusterLevelConfigForNativeReplication.java

```diff
@@ -97,4 +105,30 @@ public void testClusterLevelNativeReplicationConfigForNewStores() {
           "dc-hybrid");
     });
   }
+
+  @Test(timeOut = TEST_TIMEOUT)
+  public void testConvertHybridDuringPushjob() {
+    String storeName = Utils.getUniqueString("test-store");
+    parentControllerClient.createNewStore(storeName, "test-owner", "\"string\"", "\"string\"");
+    parentControllerClient.requestTopicForWrites(
```

Comment:
Thanks for adding the test! We didn't kill this push; would it affect other tests in the same file, since we are using the same store name here.

Reply:
Sure, good catch